src/lib/journey/callbacks/choice/choice.svelte

```diff
@@ -37,13 +42,26 @@
   }
 </script>
 
-<!-- TODO: Needs a localization strategy -->
-<Select
-  {firstInvalidInput}
-  defaultOption={defaultChoice}
-  isRequired={false}
-  key={inputName}
-  label={prompt}
-  onChange={setValue}
-  options={choiceOptions}
-/>
+{#if displayType === 'select'}
+  <!-- TODO: Needs a localization strategy -->
+  <Select
+    {firstInvalidInput}
+    defaultOption={defaultChoice}
+    isRequired={false}
+    key={inputName}
+    label={prompt}
+    onChange={setValue}
+    options={choiceOptions}
+  />
+{:else}
+  <Radio
+    {firstInvalidInput}
+    defaultOption={Number(defaultChoice)}
+    isRequired={false}
+    key={inputName}
+    message={prompt}
```

Comment:
I found that the formatting of Radio is odd, with the choices first, followed by the prompt. It looks like this is intentional, but I'm not sure why. @cerebrl are you able to explain a little about this? Personally, I think it should be the other way round, but want to understand why it is how it is now. See screenshot below for example:

![Screenshot 2022-10-05 at 18 56 19](https://user-images.githubusercontent.com/30521218/194129111-9145dafd-cc37-44bc-a5fe-9afd6e1a06cc.png)

Reply:
Ah, this is a good callout. There are two things here:

1. You're using `message` prop for displaying the `prompt`, this is why it's displayed below
2. Both the standard and the animated radio don't have "group" label, which is needed here to have something describing the group of radios available for selection.

For (1) above, this is for adding informational or error messages to an input, like input requirements or validation errors. For (2), we need to actually add this to the underlying component. Since the components already have the concept of a label for the input (the "Yes" and "No" in the above screenshot), we need to add a "group label" to the component at the composition level.

So, for `animated.svelte` and `standard.svelte`, let's add a "grouping" header, so to speak, as an exported prop that you can use to pass in the `prompt` property. I'm most familiar with using `fieldset` for doing this kind of grouping, but we may have other options. I'm mostly concerned with the accessibility of our choice. You can read more about it here: https://www.w3.org/WAI/tutorials/forms/grouping/.

You can use basic tailwind utility classes to control the styling: https://tailwindcss.com/docs/utility-first. You don't have to worry about adding a custom class to the theme for this right now.

Reply:
Understood, I think I know what I need to do for this. I'll have a go at this and hopefully push my changes later today. Thanks for the guidance.

Reply:
I've implemented a grouping header as per w3 above - @cerebrl please take another look. Thanks.

Reply:
This changes look good. I have some comments on naming, but I'll resolve this conversation and add my specific comments inline there.